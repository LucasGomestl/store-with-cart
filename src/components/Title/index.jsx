import React from "react";
import { StyledTitle } from "./styles";

const Title = ({ title }) => {
  return <StyledTitle data-testid="title">{title}</StyledTitle>;
};

export default Title;
