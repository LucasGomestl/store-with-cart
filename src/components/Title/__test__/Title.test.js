import React from "react";
import Title from "../index";
import { render } from "@testing-library/react";

describe("Title component test", () => {
  it("Should render Title", () => {
    const { container } = render(<Title title="My Title" />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
