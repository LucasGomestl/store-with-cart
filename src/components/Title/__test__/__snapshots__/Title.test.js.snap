// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component test Should render Title 1`] = `
<h1
  class="sc-AxjAm lahjEK"
  data-testid="title"
>
  My Title
</h1>
`;
